platform/pensando/sonic-platform-modules-dpu/dpu/service/dpu-dhcp-renewal.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=dpu dhcp ip renewal
```

Comment:
@prgeor , can you check why this is needed per dpu vendor? this should be generic?

Reply:
@shanshri could you please explain why we need this instead of the generic approach https://github.com/Pterosaur/SONiC/blob/master/doc/smart-switch/ip-address-assigment/smart-switch-ip-address-assignment.md